tests/bug-detectors/ssrf/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "jazzerjs-ssrf-tests",
+	"version": "1.0.0",
+	"description": "Tests for the ssrf bug detector",
+	"scripts": {
+		"test": "jest",
+		"fuzz": "JAZZER_FUZZ=1 jest"
+	},
+	"devDependencies": {
+		"@jazzer.js/jest-runner": "file:../../packages/jest-runner",
```

Comment:
There are no jest tests in this project, right? Remove all dependencies?

Reply:
I extended the helper and added some Jest tests.